test/test_utilities.jl

```diff
@@ -171,6 +171,7 @@ function test_show()
     @test curvature(p) == Convex.ConvexVexity()
     @test sprint(show, p) == """
     Problem statistics
+      problem is DCP         : true
```

Comment:
@ericphanson what do you think of this?

Reply:
Could be helpful? I don’t feel strongly either way